eahub/base/views.py

```diff
@@ -42,10 +42,11 @@ def getGroupsData():
     rows = Group.objects.all()
     map_data = ''.join([
         '{' +
-            'lat: {lat}, lng: {lon}, label:"{name}", path: "{path}"'.format(
+            'lat: {lat}, lng: {lon}, label:"{name}", active:"{active}", path: "{path}"'.format(
```

Comment:
Shouldn't this be a boolean instead of a string?

Reply:
This is sent to Javascript, and Javascript can't read Python's Booleans (throws error "True is not defined" in console)